commitlint.config.js

```diff
@@ -3,7 +3,6 @@ module.exports = {
   rules: {
     'header-max-length': [1, 'always', 50],
     'body-max-line-length': [2, 'always', 72],
-    'footer-max-line-length': [2, 'always', 72],
-    'type-enum': [2, 'always', ['feat', 'fix', 'docs', 'style', 'refactor', 'test', 'revert', 'chore']]
```

Comment:
Why'd you remove the commit type checking? Seems like one of the big reasons to even check commits...

Reply:
I just removed the fact that I was _overriding_ [the default](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional#type-enum).